From: Michal Kubecek <mkubecek@suse.cz>
Date: Sat, 8 Oct 2016 00:32:42 +0200
Subject: kabi: hide name change of napi_gro_cb::udp_mark
Patch-mainline: Never, kabi workaround
References: bsc#1001486

The fix for bsc#1001486 renames member udp_mark of struct sk_buff to
encap_mark as it's no longer used only for UDP but also for all
tunneling protocols. To preserve kABI, hide this change from genksyms.

Signed-off-by: Michal Kubecek <mkubecek@suse.cz>
---
 include/linux/netdevice.h | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/include/linux/netdevice.h b/include/linux/netdevice.h
index 9ee0b7d5affc..71a271a57e6d 100644
--- a/include/linux/netdevice.h
+++ b/include/linux/netdevice.h
@@ -2059,7 +2059,11 @@ struct napi_gro_cb {
 	u8	same_flow:1;
 
 	/* Used in tunnel GRO receive */
+#ifdef __GENKSYMS__
+	u8	udp_mark:1;
+#else
 	u8	encap_mark:1;
+#endif
 
 	/* GRO checksum is valid */
 	u8	csum_valid:1;
-- 
2.10.0

